_topic_map.yml

```diff
@@ -620,18 +620,27 @@ Topics:
         File: ruby_on_rails
       - Name: Setting Up a Nexus Mirror
         File: maven_tutorial
-  - Name: Opening a Remote Shell to Containers
-    File: ssh_environment
-  - Name: Templates
-    File: templates
-  - Name: Service Accounts
-    File: service_accounts
   - Name: Builds
-    File: builds
-  - Name: Managing Images
-    File: managing_images
-  - Name: Quotas and Limit Ranges
-    File: compute_resources
+    Dir: builds
+    Topics:
+      - Name: How Builds Work
+        File: how_builds_work
+      - Name: Basic Build Operations
+        File: basic_build_operations
+      - Name: Build Inputs and Output
+        File: build_inputs_output
+      - Name: Build Strategies
+        File: build_strategies
+      - Name: Triggering Builds
+        File: triggering_builds
+      - Name: Secrets During Builds
```

Comment:
Yeah "Secrets During Builds" is weird. That's what it was originally, was trying to come up with something different... Placeholder for now. I bet it'll just end up back at "Using Secrets During Builds" ultimately.